Composer/packages/server/src/models/utilities/parser.ts

```diff
@@ -64,3 +70,18 @@ export async function parseQnAContent(url: string, multiTurn: boolean) {
 
   return await importQnAFromUrl(builder, url, subscriptionKey, multiTurn);
 }
+
+// subscriptionKey from selection.
+export async function importQnAContentFromQnAMakerPortal(endpoint: string, kbId: string, subscriptionKey: string) {
+  const cognitiveServicesCredentials = new CognitiveServicesCredentials(subscriptionKey);
+  const resourceClient = new QnAMakerClient(cognitiveServicesCredentials, endpoint);
+
+  // 'Test' or 'Prod'
+  const result = await resourceClient.knowledgebase.download(kbId, 'Test');
```

Comment:
@boydc2014 Thoughts here? https://docs.microsoft.com/en-us/javascript/api/@azure/cognitiveservices-qnamaker/knowledgebase?view=azure-node-latest#download_string__Models_EnvironmentType__KnowledgebaseDownloadOptionalParams__ServiceCallback_QnADocumentsDTO__

It looks like we can have the test Index and then we can promote the KB to the prod index.

Reply:
I noticed that if a KB have not been published, only 'Test' can download, so here set 'Test' to default to cover more case.